project-koku/koku
# [COST-4514] Upgrade to django 4.0

## Jira Ticket

[COST-4514](https://issues.redhat.com/browse/COST-4514)

## Description

This change upgrades Django to 4.0 and DRF to 3.14 versions

## Testing

1. Checkout Branch
2. Restart Koku
3. Load test data
4. See that ingestion and summary complete successfully

## Notes


...
